examples/ramen_rec/data/ramen_ratings_to_kg.py

```diff
@@ -1,13 +1,15 @@
 import rdflib
 from rdflib.namespace import RDF, RDFS, XSD
 import csv
+import random
 
 
 def convert():
```

Comment:
Fine to do this for now. If you need something like this longer-term I would probably have `@dataclass`'s with a `to_rdf` method. See https://github.com/tetherless-world/mcs-benchmark-data/tree/master/mcs_benchmark_data/models.